<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-10T13:56:19Z" name="Default suite" finished-at="2023-04-10T13:56:21Z" duration-ms="1669">
    <groups>
    </groups>
    <test started-at="2023-04-10T13:56:19Z" name="Default test" finished-at="2023-04-10T13:56:21Z" duration-ms="1669">
      <class name="com.qa.tests.GetAPITest">
        <test-method is-config="true" signature="setUP()[pri:0, instance:com.qa.tests.GetAPITest@4066c471]" started-at="2023-04-10T19:26:19Z" name="setUP" finished-at="2023-04-10T19:26:19Z" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUP -->
        <test-method signature="getTest()[pri:0, instance:com.qa.tests.GetAPITest@4066c471]" started-at="2023-04-10T19:26:19Z" name="getTest" finished-at="2023-04-10T19:26:21Z" duration-ms="1652" status="FAIL">
          <exception class="org.json.JSONException">
            <message>
              <![CDATA[A JSONObject text must begin with '{' at 1 [character 2 line 1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]
	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
	at org.json.JSONObject.<init>(JSONObject.java:195)
	at org.json.JSONObject.<init>(JSONObject.java:319)
	at com.qa.client.RestClient.get(RestClient.java:33)
	at com.qa.tests.GetAPITest.getTest(GetAPITest.java:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.json.JSONException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTest -->
      </class> <!-- com.qa.tests.GetAPITest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
